frontend/src/components/Shared/CrossingListItem/Location/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './Location.css';
+
+class Location extends React.Component {
+  render() {
+    const { crossing } = this.props;
```

Comment:
I noticed a few places where the entire crossing object is being passed in and a few where only the props we need for the component are passed in. Do you have a preference one way or the other that makes consistency worth going for?

Reply:
Discussed IRL. If its not crossing specific, put it in /components/Shared/.